<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="deviceFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>351</width>
      <height>361</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {border: 2px solid black; background: qlineargradient(spread:pad, x1:0.762, y1:0, x2:0.51269, y2:1, stop:0 rgba(255, 241, 199, 255), stop:0.994924 rgb(220, 204, 129, 255)) }
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="blueWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(0, 0, 0)</string>
     </property>
    </widget>
    <widget class="QWidget" name="greenWidget" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(0, 0, 0)</string>
     </property>
    </widget>
    <widget class="QWidget" name="redWidget" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(0, 0, 0)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menuButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(255, 241, 199, 255);
image: url(:/menu.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>241</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QLabel" name="newSessionLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent; border: none;</string>
       </property>
       <property name="text">
        <string>NEW SESSION</string>
       </property>
      </widget>
      <widget class="QLabel" name="sessionLogLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent; border: none;</string>
       </property>
       <property name="text">
        <string>SESSION LOG</string>
       </property>
      </widget>
      <widget class="QLabel" name="dateLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent; border: none;</string>
       </property>
       <property name="text">
        <string>TIME AND DATE</string>
       </property>
      </widget>
      <widget class="QLabel" name="testLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent; border: none;</string>
       </property>
       <property name="text">
        <string>TEST UI</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>211</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="timerLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>48</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent; border: none;</string>
       </property>
       <property name="text">
        <string>0:00</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>8</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2024</year>
         <month>4</month>
         <day>12</day>
        </datetime>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>8</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2024</year>
         <month>4</month>
         <day>12</day>
        </datetime>
       </property>
       <property name="date">
        <date>
         <year>2024</year>
         <month>4</month>
         <day>12</day>
        </date>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent; border: none;</string>
       </property>
       <property name="text">
        <string>Selected The Date and Time:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent; border: none;</string>
       </property>
       <property name="text">
        <string>View Logs:</string>
       </property>
      </widget>
      <widget class="QListView" name="listView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>211</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_6">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent; border: none;</string>
       </property>
       <property name="text">
        <string>Test UI</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>221</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid black;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent; border: none; image: url(:/power.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="batteryLabel">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>330</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent; border: 1px ;</string>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
    <widget class="QLabel" name="batteryIconLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>320</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent; border: 1px ; image: url(:/battery.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lowBatteryLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent; border: 1px ; color: red</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(36, 31, 49); color: white</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>7</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2024</year>
       <month>4</month>
       <day>12</day>
      </datetime>
     </property>
     <property name="date">
      <date>
       <year>2024</year>
       <month>4</month>
       <day>12</day>
      </date>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::YearSection</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/pause.png); 
background-color: rgb(222, 221, 218);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pause.png</normaloff>pause.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="autoRepeatDelay">
      <number>300</number>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>250</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/play.png); 
background-color: rgb(222, 221, 218);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>play.png</normaloff>play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>250</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> image: url(:/stop.png); 
background-color: rgb(222, 221, 218);
 </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>stop.png</normaloff>stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="upButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>110</y>
       <width>39</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/up.png); 
background-color: rgb(222, 221, 218);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="downButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>39</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> image: url(:/down.png); 
background-color: rgb(222, 221, 218);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="selectButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>39</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/enter.png); 
background-color: rgb(222, 221, 218);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="auxLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent; border: 1px ; color: green</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="charger">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(222, 221, 218); image: url(:/charger.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>501</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pc.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TEST BUTTON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="auxcord">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(222, 221, 218); image: url(:/aux1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; border: none;</string>
    </property>
    <property name="text">
     <string>Headsets</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; border: none;</string>
    </property>
    <property name="text">
     <string>Charge</string>
    </property>
   </widget>
   <widget class="QLabel" name="pcLabel1">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNeureset">
    <property name="title">
     <string>Neureset</string>
    </property>
   </widget>
   <addaction name="menuNeureset"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
